react/tic-tac-toe/src/app/screens/Game/components/Matches/index.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+
+import MatchesService from '../../../../../services/MatchesService';
+
+import styles from './styles.module.scss';
+
+const Spinner = require('react-spinkit');
+
+const getWinnerClass = isWinner => (isWinner ? styles.winner : '');
+
+class Matches extends Component {
+  state = {
+    data: [],
+    loading: true
+  };
+
+  componentDidMount() {
+    const match = MatchesService.getMatches();
+    match.then(result => {
+      this.setState({ data: result.data, loading: false });
+    });
+  }
+
+  render() {
+    return (
+      <div>
```

Comment:
If `<div>` doesn't have a class or style but you do really need it, I would suggest using `<Fragment>`
It has some advantages, such as:

> It’s a tiny bit faster and has less memory usage (no need to create an extra DOM node). This only has a real benefit on very large and/or deep trees, but application performance often suffers from death by a thousand cuts. This is one cut less.
> Some CSS mechanisms like Flexbox and CSS Grid have a special parent-child relationship, and adding divs in the middle makes it hard to keep the desired layout while extracting logical components